pkgs/stdenv/generic/setup.sh

```diff
@@ -765,10 +792,17 @@ installPhase() {
     fi
 
     installTargets=${installTargets:-install}
-    echo "install flags: $installTargets $makeFlags ${makeFlagsArray[@]} $installFlags ${installFlagsArray[@]}"
-    make ${makefile:+-f $makefile} $installTargets \
+
+    # shellcheck disable=SC2086
+    local flagsArray=( $installTargets \
         $makeFlags "${makeFlagsArray[@]}" \
-        $installFlags "${installFlagsArray[@]}"
+        $installFlags "${installFlagsArray[@]}")
+
+    printf 'install flags:'
```

Comment:
@dezgeg it hash to be:

1. print the name of the flags
2. repeat the format string (*un*quoted space + quoted arg) for each argument
3. close the line

printf repeats the format string until all the args are gone, so it is essential the repeated format string contain one a single substitution.